docs/modules/general/introduction.mdx

```diff
@@ -6,24 +6,34 @@ title: Introduction
 ### History
 
 PGM originally started in 2012 as a single game plugin specifically built to run one map, Airship Battle.
-This plugin could manage the match and reset the map easily at the end of the game.
+This plugin could manage the match and reset the map easily at the end of the game, which was unique at the time as there were no comprehensive Bukkit plugins that could manage PvP matches.
 Other maps were created later, but these all had to be hard-coded into the plugin.
 This was cumbersome and complicated, and there needed to be a new way to quickly program maps into the plugin using an easy to understand language and syntax.
 This problem would later meet its solution, the PvP Game Manager.
 Players unknown to Java programming could now create their own games and matches using an XML file that PGM takes as an input.
-As of now, there are over 1000 maps made for PGM!
+This was the signature plugin that was exclusive to a major Minecraft server called Overcast Network (also known as oc.tc).
+After the server's clousure, it was open sourced for the community to enjoy.
+A new server called Stratus Network became the lead maintainer of PGM, adding new features to it, but later on they became unable to maintain the plugin as it became large and messy.
```

Comment:
Don't think this is really necessary as 1.8 PGM is an early fork from the original Overcast PGM code

Reply:
Do you think this part should be removed?

Reply:
Yea, I would refrain from mentioning other servers (even Overcast, but in this case I think mentioning OCN makes sense)

Reply:
Fair enough.